dialog/file.go

```diff
@@ -181,10 +184,36 @@ func (f *fileDialog) makeUI() fyne.CanvasObject {
 
 	favorites := f.loadFavorites()
 
-	favoritesGroup := widget.NewGroup("Favorites", favorites...)
-	return fyne.NewContainerWithLayout(layout.NewBorderLayout(header, footer, favoritesGroup, nil),
-		favoritesGroup, header, footer, body)
+	favoritesGroup := widget.NewVScrollContainer(widget.NewGroup("Favorites", favorites...))
```

Comment:
It might be outside the scope of this PR, but given that we now are relying on scrolling, it probably makes sense to use a `widget.List` instead, for better performance.

Reply:
Need more experienced brains than mine to answer that properly, but I would have thought the favourites list is not a huge performance hit as I would expect it to be short.

Reply:
Sure. You can probably just ignore this comment. It is definitely outside the scope of this PR.

Reply:
I don’t think we should start using List in every time that there is a scrolling VBox, the semantics are different. It would mean introducing selection. 
And unless you’re scrolling 100s of items it likely makes little difference. Here we will have somewhere between 5 and 30 elements only.